src/db.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const { Sequelize } = require('sequelize');
+
+const sequelize = new Sequelize(
+  process.env.PG_NAME,
+  process.env.PG_USER,
+  process.env.PG_PASSWORD,
+  {
+    dialect: 'postgres',
+    host: 'localhost',
```

Comment:
Replace the hardcoded 'localhost' with 'process.env.PG_HOST' to make the host configurable.